doc/administrator_guide.md

```diff
@@ -5,32 +5,24 @@ draft: false
 wieght: 40
 description: "Deploying, maintaining, and upgrading a KCIDB service installation"
 ---
-Kcidb infrastructure is mostly based on Google Cloud services at the moment:
-
-    === Hosts ===  ======================= Google Cloud Project ========================
-
-    ~ Webserver ~                                                    ~~~~ BigQuery ~~~~~
-    kcidb-grafana <-------------------------------------------------  .. kcidb_XX ..
-                                                                     :   checkouts  :
-    ~~ Client ~~~                                                    :   builds     :
-    kcidb-query <--------------------------------------------------- :   tests      :
-                                                                      ''''''''''''''
-                    ~~ Pub/Sub ~~       ~~~~ Cloud Functions ~~~~            ^
-                    kcidb_trigger ----------.                                |
-                                            v                                |
-    kcidb-submit -> kcidb_new --------> kcidb_load_queue --------------------'
-                                            |
-                          .-----------------'
-                          v                                          ~~~~ Firestore ~~~~
-                    kcidb_updated ----> kcidb_spool_notifications -> notifications
-                                                                           |
-                                                   .-----------------------'
-                                                   |
-                                                   v                 ~ Secret Manager ~~
-                                        kcidb_send_notification <--- kcidb_smtp_password
-                                                   |
-                                                   |                 ~~~~~~ GMail ~~~~~~
-                                                   `---------------> bot@kernelci.org
+```mermaid
+graph TD
+  Webserver["Webserver"] --> BigQuery["BigQuery"]
+  Client["~~ Client ~~"] --> BigQuery
```

Comment:
The `~~` around the `Client` were just part of ASCII art :joy: We can drop the label here as well.

Reply:
ok